demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -1453,6 +1438,51 @@ export class DemoMeetingApp
     this.initContentShareDropDownItems();
   }
 
+  async setupEventReporter(configuration: MeetingSessionConfiguration): Promise<EventReporter> {
+    let eventReporter: EventReporter;
+    const ingestionURL = configuration.urls.eventIngestionURL;
+    if (!ingestionURL) {
+      return eventReporter;
+    }
+    if (this.enableEventReporting) {
+      const eventReportingLogger = new ConsoleLogger('SDKEventIngestion', LogLevel.INFO);
+      const meetingEventClientConfig = new MeetingEventsClientConfiguration(
+        configuration.meetingId,
+        configuration.credentials.attendeeId,
+        configuration.credentials.joinToken
+      );
+      const eventIngestionConfiguration = new EventIngestionConfiguration(
+        meetingEventClientConfig,
+        ingestionURL
+      );
+      if (this.isLocalHost()) {
+        eventReporter = new DefaultMeetingEventReporter(eventIngestionConfiguration, eventReportingLogger);
+      } else {
+        await this.createLogStream(configuration, 'create_browser_event_ingestion_log_stream');
+        const eventReportingPOSTLogger = new MeetingSessionPOSTLogger(
+          'SDKEventIngestion',
+          configuration,
+          DemoMeetingApp.LOGGER_BATCH_SIZE,
+          DemoMeetingApp.LOGGER_INTERVAL_MS,
+          `${DemoMeetingApp.BASE_URL}log_event_ingestion`,
+          LogLevel.DEBUG
+        );
+        const multiEventReportingLogger = new MultiLogger(
+          eventReportingLogger,
+          eventReportingPOSTLogger,
+        );
+        eventReporter = new DefaultMeetingEventReporter(eventIngestionConfiguration, multiEventReportingLogger);
+      }
+    } else {
```

Comment:
this could move up also right? If event reporting is not enabled then return noOp reporter and quit right away?

Reply:
Yes, can avoid the else block here :) Looks more cleaner :) Thanks, have updated it.